tsinfer/formats.py

```diff
@@ -1185,9 +1204,11 @@ def from_tree_sequence(cls, ts, use_times=True, use_metadata=True, **kwargs):
                     samples_metadata=[sample_metadata],
                 )
         for v in ts.variants():
-            variant_time = None
-            if use_times and len(v.site.mutations) == 1:
-                variant_time = ts.node(v.site.mutations[0].node).time
+            variant_time = self.TIME_UNSPECIFIED
+            if use_times:
+                variant_time = self.TIME_MEANINGLESS
```

Comment:
To be clear, I don't think we should have TIME_MEANINGLESS at all.

Reply:
See my [comment above](https://github.com/tskit-dev/tsinfer/pull/254#discussion_r399219938). I think it is distinct and useful...